tools/test/audio/sof_test_perf.m

```diff
@@ -29,7 +29,8 @@
 
 %% Plotting
 test.plot_close_windows = 1;    % Workaround for visible windows if Octave hangs
-test.plot_visible = 'off';      % Use off for batch tests and on for interactive
+% test.plot_visible = 'off';      % Use off for batch tests and on for interactive
+test.plot_visible = 'on';      % Use off for batch tests and on for interactive
```

Comment:
The defaults should be for batch. But there could be a nicer way to enable plots open for interactive session. Do you have ideas?

Reply:
To enable plots open for interactive sessions, I will reverse the present change to its original state and submit a new pull request.